tests/test_utils.py

```diff
@@ -81,6 +81,10 @@ def test_merge_responses_one_page_results(
     one_page_analyse_result: AnalyzeResult,
 ) -> None:
     """Test that the responses are merged correctly."""
+    one_page_analyse_result.documents = ["test_document"]
+    one_page_analyse_result.languages = ["test_language"]
+    one_page_analyse_result.styles = ["test_style"]
```

Comment:
I think you're gonna need to explain the rationale behind these changes in a comment just to explain what these changes allow us to verify. It's not exactly clear to me now & future-you will thank you :grin: